data/common.yaml

```diff
@@ -1,2 +1,2 @@
-cvmfs::repo_incude_pkgs: ~
+cvmfs::repo_includepkgs: ~
```

Comment:
I think that bugfix should be a dedicated PR? It's not really related to adding Debian/Ubuntu support

Reply:
https://github.com/voxpupuli/puppet-cvmfs/pull/126